extensions/px-to-rem-converter/src/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Action, ActionPanel, List } from "@raycast/api";
+import React, { useState } from "react";
+
+const BASE_FONT_PIXELS = 16;
```

Comment:
Maybe make this a [preference](https://developers.raycast.com/api-reference/preferences)?

Reply:
Good idea! 👍🏼